imagersite/imager_images/tests.py

```diff
@@ -0,0 +1,49 @@
+
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.test import TestCase, Client
+from django.urls import reverse_lazy
+from bs4 import BeautifulSoup
```

Comment:
This line causes me an error when I try and run your tests. You need to update your requirements.pip so that I can have all the dependencies installed.